mrobosub_hal/src/dvl_publisher.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import socket
+
+import rospy
+from mrobosub_msgs.msg import Dvl
+
+
+# todo: parameterize this in the launch file
+UDP_IP = "0.0.0.0"
+UDP_PORT = 27000
+
+
+class DVLPublisher():
+    def __init__(self):
+        rospy.init_node("dvl_publisher")
+        self.pub = rospy.Publisher('/dvl/raw_dvl', Dvl, queue_size=10)
```

Comment:
I think since this is data from a sensor we want the queue size to be 1 as we always want the latest sensor information

Reply:
Done